### PR TITLE
Docs v3

### PR DESCRIPTION
**Pull request by alexand3rwilke**
## Summary
- light docs page for the new website
- overview of key parts
- added xbi docs from xbi repo
- added collator part from previous docs

### CODE REVIEW

File: docs/main/static/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      http-equiv="refresh"
+      content="0; url=docs/0"
+    />
+    <script type="text/javascript">
+      window.location.href = 'docs/';
+    </script>
+    <title>Your Site Title Here</title>
```
Comment (MaciejBaj) [inline docs/main/static/index.html:12]: ?

File: docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+|72|0.2|0.5|0.8|1.1|4.40 %|
+
+## Gas Fees
+
+| | Collators | Executors | Attesters | Authors | Treasury | Users |
+|---|---|---|---|---|---|---|
+| Any TX| 80% | N/A | N/A | N/A | 20% | 0% |
+| Contracts Call TX | 10% | N/A | N/A | 80% | 10% | 0% |
+
+
```
Comment (MaciejBaj) [inline docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md:33]: single new line ending to all docs

Comment (alexand3rwilke) [reply docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md]: [1131805](https://github.com/t3rn/t3rn/pull/519/commits/1131805fb04d61c4bd875e7bdb7929a8080f8cb5)

File: docs/main/static/index.html (hunk 1 above)
Comment (alexand3rwilke) [reply docs/main/static/index.html]: [5a1681b](https://github.com/t3rn/t3rn/pull/519/commits/5a1681ba7811b12d84768f12905a5d4d1d72ee4a)

**Review by MaciejBaj — DISMISSED** [review]

**Review by petscheit — APPROVED** [review]
LGTM

**Pull request merged**